### ISSUE TITLE
Hyperparameters for training SAC algorithm on half-cheetah-goal environment

### ISSUE THREAD

**Issue by xuyangthu88** [Princeton-RL/contrastive-successor-features#2]
--env half_cheetah \
                        --max_path_length 200 \
                        --seed $seed \
                        --traj_batch_size 8 \
                        --n_parallel 8 \
                        --normalizer_type preset \
                        --trans_optimization_epochs 50 \
                        --n_epochs_per_log 100 \
                        --n_epochs_per_eval 1000 \
                        --n_epochs_per_save 10000 \
                        --sac_max_buffer_size 1000000 \
                        --algo sac \
                        --discrete 1 \
                        --dim_option 2 \
                        --goal_range 100 \
                        --eval_goal_metrics 1 \
                        --turn_off_dones 1
 
I used the above parameters to train the sac algorithm. The hyperparameters are the same as in metra_half_cheetah. It can complete the half_cheetah task well, but cannot complete the half_cheetah_goal task. 

So what hyperparameters should I choose to train SAC algorithm in half_cheetah_goal task?

**Comment by jens321** [comment]
Hi @xuyangthu88, thanks for your interest in our work! 

Our repo does not directly support training SAC on the half_cheetah_goal task (although it probably only requires a few minor modifications). Instead, the intended use of our repo is as follows:
1. Pretrain a skill-conditioned policy using an unsupervised RL method with any of the scripts here: https://github.com/Princeton-RL/contrastive-successor-features/tree/main/scripts/pretrain. Note that these policies will not be trained to do any specific task, but rather produce skills that are generally useful in the environment.
2. Train a _**hierarchical**_ control policy using either SAC or PPO using any of the scripts here: https://github.com/Princeton-RL/contrastive-successor-features/tree/main/scripts/hierarchical_control. If you were interested in training a hierarchical policy for half_cheetah_goal using another pretrained skill-conditioned policy, you can use this script: https://github.com/Princeton-RL/contrastive-successor-features/blob/main/scripts/hierarchical_control/half_cheetah_goal.sh

Hope that helps and let us know how it goes!
